test/selenium/wms_test.js

```diff
@@ -0,0 +1,46 @@
+// WMS test using browserstack
+
+var webdriver = require('browserstack-webdriver');
+var assert = require('assert');
+
+var QUERYSTRING_WMS = "WMS%7C%7CAGNES%7C%7Chttp:%2F%2Fwms.geo.admin.ch%2F%7C%7Cch.swisstopo.fixpunkte-agnes";
+
+var runTest = function(cap, driver, target) {
+  //We maximize our window to be sure to be in full resolution
+  driver.manage().window().maximize();
+  // Goto the travis deployed site.
+  driver.get(target + '/?lang=de');
+  //wait until topics related stuff is loaded. We know this when catalog is there
+  driver.findElement(webdriver.By.xpath("//a[contains(text(), 'Grundlagen und Planung')]"));
+
+  // Click on "Werkzeuge"
+  driver.findElement(webdriver.By.xpath("//a[@id='toolsHeading']")).click();
+  // Click on "WMS Import"
+  driver.findElement(webdriver.By.xpath("//*[contains(text(), 'WMS Import')]")).click();
+  // Click on the URL input field
+  driver.findElement(webdriver.By.xpath("//*[@id='import-wms-popup']//input[@placeholder='URL']")).click();
+  // Write URL of the chosen WMS
+  driver.findElement(webdriver.By.xpath("//*[@id='import-wms-popup']//input[@placeholder='URL']")).sendKeys('http://wms.geo.admin.ch/');
+  // Click on "Verbinden"
+  driver.findElement(webdriver.By.xpath("//*[@id='import-wms-popup']//button[contains(text(),'Verbinden')]")).click();
+  // Click on "AGNES"
+  driver.findElement(webdriver.By.xpath("//*[@id='import-wms-popup']//div[contains(text(),'AGNES')]")).click();
+  // Click on "Layer hinzufügen"
+  driver.findElement(webdriver.By.xpath("//*[@id='import-wms-popup']//button[contains(text(),'Layer hinzufügen')]")).click();
+  // Accept the alert
+  driver.switchTo().alert().accept();
```

Comment:
The alert is the way we assess the user that the layer is now loaded. I do not like alerts but this won't disapears with 3rd party data agreement.